security/rbac/docs/index.rst

```diff
@@ -0,0 +1,460 @@
+
+Docker
+======
+
+This method of running the demo is for users who have Docker.
+This demo setup includes:
+
+-  |zk|
+-  Kafka with MDS, connected to the OpenLDAP
+-  |sr|
+-  KSQL
+-  |kconnect-long|
+-  |crest|
+-  |c3|
+-  OpenLDAP
+
+Prerequisites
+-------------
+
+-  Docker
+-  ``zookeeper-shell`` must be on your ``PATH``
+-  :ref:`Confluent CLI <cli-install>`:
+   |confluent-cli| must be installed on your machine, version
+   ``v0.127.0`` or higher (note: as of CP 5.3, the |confluent-cli| is a separate
+   :ref:`download <cli-install>`
+
+
+Image Versions
+--------------
+
+-  You can use production or pre-production images. This is configured
+   via environment variables ``PREFIX`` and ``TAG``.
+
+   -  ``PREFIX`` is appended before the actual image name, before ``/``
+   -  ``TAG`` is a docker tag, appended after the ``:``
+   -  E.g. with ``PREFIX=confluentinc`` and ``TAG=5.3.1``, kafka will
+      use the following image: ``confluentinc/cp-server:5.3.1``
+   -  If these variables are not set in the shell, they will be read
+      from the ``.env`` file. Shell variables override whatever is set
+      in the ``.env`` file
+   -  You can also edit ``.env`` file directly
+   -  This means all images would use the same tag and prefix. If you
+      need to customize this behavior, edit the ``docker-compose.yml``
+      file
+
+Run the demo
+------------
+
+#. Clone the `confluentinc/examples <https://github.com/confluentinc/examples>`__ repository from GitHub and check out the :litwithvars:`|release|-post` branch.
+
+   .. codewithvars:: bash
+
+       git clone git@github.com:confluentinc/examples.git
+       cd examples
+       git checkout |release_post_branch|
+
+#. Navigate to ``security/rbac/scripts`` directory.
+
+   .. codewithvars:: bash
+
+       cd security/rbac/rbac-docker
+
+#. To start |cp|, run
+
+   .. code:: bash
+
+      ./confluent-start.sh
+
+You can optionally pass in where ``-p project-name`` to name the
+docker-compose project, otherwise it defaults to ``rbac``. You can use
+standard docker-compose commands like this listing all containers:
+
+   .. code:: bash
+
+      docker-compose -p rbac ps
+
+or tail |c3| logs:
+
+   .. code:: bash
+
+      docker-compose -p rbac logs --t 200 -f control-center
+   
+The Kafka broker is available at ``localhost:9094`` (not ``localhost::9092``).
+
+=============== ==================
+Service         Host:Port
+=============== ==================
+Kafka           ``localhost:9094``
+MDS             ``localhost:8090``
+C3              ``localhost:9021``
+Connect         ``localhost:8083``
+KSQL            ``localhost:8088``
+OpenLDAP        ``localhost:389``
+Schema Registry ``localhost:8081``
+=============== ==================
+
+Grant Rolebindings
+~~~~~~~~~~~~~~~~~~
+
+#. Login to CLI as ``professor:professor`` as a super user to grant initial role bindings
+
+   .. code:: bash
+
+      confluent login --url http://localhost:8090
+
+#. Set ``KAFKA_CLUSTER_ID``
```

Comment:
numbering and indentation is off 
![image](https://user-images.githubusercontent.com/11722533/68430861-c106e280-0165-11ea-9833-6f593531aa8f.png)

Reply:
Fixing indentation above hopefully fixes this, I will check.